syntax = "proto3";
package lichtenstein.protocol;

import "google/protobuf/any.proto";

/**
 * Contains the response part of the challenge/response protocol.
 */
message AuthResponse {
    // method-specific response data
    google.protobuf.Any payload = 1;
}